applications/CoSimulationApplication/custom_external_libraries/CoSimIO/CMakeLists.txt

```diff
@@ -85,7 +85,7 @@ if(MSVC)
     endif()
 
 elseif(${CMAKE_COMPILER_IS_GNUCXX})
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17")
```

Comment:
this is a small standalone lib, this should not be changed I would say

Reply:
does that mean that ```CoSimIO``` will use a different ```libstdc++``` than the rest of Kratos? Is that safe?

Reply:
Yes it is safe, its the same scenario as linking with a fotran module

Reply:
Same as for any other system lib no?
Isnt that sorta garantied through ABI stability?

Reply:
Well C++11 broke the ABI compatibility for the strings with old standards (remember the infamous `_GLIBCXX_USE_CXX11_ABI`). It is expectable, but not 100% guaranteed

Reply:
To be honest, I don't know much about compatibility and maintenance. I just noticed that all my binaries are updated whenever I get a new GCC version. And new C++ standards sometimes remove features / modify some signatures.